Added template for pull requests #109196 (#116779)
AdguardTeam/AdguardFilters: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md

### Please include a summary of the change and which issue is fixed.
### If the related issue does not exist in our repository, please create it before making pool request.
### It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/en/technical-support/reporting-tool) instead of creating an issue on GitHub directly.
### Please note, that we verify every pull request manually, so it may take time to apply it.

---

### Prerequisites
##### To avoid invalid pull requests, please check and confirm following checkboxes:

        
  - [ ] This is not an ad/bug report;

>   - [ ] My code follows the style guidelines and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;

Reviewer: ```suggestion
  - [ ] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
```

  - [ ] I have performed a self-review of my own changes;
  - [ ] My changes do not break web sites, apps and files structure.

> ### What what problem does pull request fix?

Reviewer 2: > What what

Maybe typo?

Author: fixed

> ##### If the problem does not fall under any category that is listed here, please contact our tech support - support@adguard.com

Reviewer: ```suggestion
##### If the problem does not fall under any category that is listed here, please contact our tech support: `support@adguard.com`
```

  - [ ] Missed ads or ad leftovers;
  - [ ] Website or app doesn't work properly;
  - [ ] AdGuard gets detected on a website;
  - [ ] Missed analytics or tracker;
  - [ ] Social media buttons — share, like, tweet, etc;
  - [ ] Annoyances — pop-ups, cookie warnings, etc.
  

> ### Which issue is fixed?

Reviewer: ```suggestion
### What issue is being fixed?
```

> ##### Enter an issue address:

Reviewer: ```suggestion
##### Enter the issue address:
```

Example: https://github.com/AdguardTeam/AdguardFilters/issues/100000

### Add your comment and screenshots
##### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located.

**0.** **DO NOT** upload screenshots with sexually explicit material on GitHub directly. Instead, upload it to third-party image hosting and post URL without preview here;

**1.** Add screenshots of the problem. You can drag and drop images or paste them from clipboard;

Use `<details> </details>` tag to hide screenshots under the spoiler; 

> **2.** Describe the issue extensively unless it is absolutely clear from the screenshot what the problem is;

Reviewer: ```suggestion
**2.** Describe the issue in detail until it is absolutely clear from the screenshot what the problem is;
```
